jupyter_foundation.md

```diff
@@ -0,0 +1,27 @@
+# Jupyter Foundation
+
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6). It exists to provide resources and strategic counsel to Project Jupyter. The charter is here (TODO: Link). We summarize relevant points from the charter below.
+
+
+## Jupyter Foundation Governing Board
```

Comment:
I unfortunately missed the last EC meetings due to travel/illness, so forgive me if this was discussed to death already. But my inclination would be to name this "something something Council" instead of "Governing Board" to keep in line with our standard naming practices for most of our decision making bodies of this type.

If there's a good reason to divert from our practice, we should encode that in the [FAQ](https://jupyter.org/governance/linux-proposal.html) (right now I see no mention of the word "Board" there).

Reply:
Just discussed it, keeping it as "Jupyter Foundation Governing Board" b/c:

1. That's the standard terminology already used by LF in many other places, and it will benefit us greatly when fundraising for this specific body to use established patterns.
2. This is really a new type of group, whose job is to govern our new fund under the 501(c)6. We've never had anything quite like that, so it is actually useful to signal its difference via naming.

This makes sense to me, resolving.